mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.test.js

```diff
@@ -22,100 +37,123 @@ describe('unit tests', () => {
         location.search = '?' + url.split('?')[1];
       },
     };
+    getMetricHistoryApi = jest.fn(() => Promise.resolve());
+    getRunApi = jest.fn(() => Promise.resolve());
+    const now = new Date().getTime();
     minimalPropsForLineChart = {
       experimentId: '1',
       runUuids: ['runUuid1', 'runUuid2'],
+      completedRunUuids: ['runUuid1', 'runUuid2'],
       metricKey: 'metric_1',
       latestMetricsByRunUuid: {
-        runUuid1: { metric_1: 100, metric_2: 200 },
-        runUuid2: { metric_1: 111, metric_2: 222 },
+        runUuid1: {
+          metric_1: { key: 'metric_1', value: 100, step: 2, timestamp: now },
+          metric_2: { key: 'metric_2', value: 111, step: 0, timestamp: now },
+        },
+        runUuid2: {
+          metric_1: { key: 'metric_1', value: 200, step: 4, timestamp: now },
+          metric_2: { key: 'metric_2', value: 222, step: -3, timestamp: now },
+        },
       },
       distinctMetricKeys: ['metric_1', 'metric_2'],
       // An array of { metricKey, history, runUuid, runDisplayName }
       metricsWithRunInfoAndHistory: [
+        // Metrics for runUuid1
         {
           metricKey: 'metric_1',
           history: [
             /* Intentionally reversed timestamp and step here for testing */
-            { key: 'metric_1', value: 100, step: 2, timestamp: 1556662044000 },
-            { key: 'metric_1', value: 50, step: 1, timestamp: 1556662043000 },
+            { key: 'metric_1', value: 100, step: 2, timestamp: now },
+            { key: 'metric_1', value: 50, step: 1, timestamp: now - 1 },
```

Comment:
Replaced hardcoded timestamps with `now` to prevent the metrics plot from considering these runs as hanging.